tests/pages/index.js

```diff
@@ -14,21 +14,22 @@ class PageObject {
 
 class CommentOnPage extends PageObject {
   body() {
-    return this.$('.comment-body').
-      text().
-      replace(/\s+/g, ' ').
-      replace(/^\s*/, '').
-      replace(/\s*$/, '');
+    return this.$('.comment-body')
+      .text()
+      .replace(/\s+/g, ' ')
+      .replace(/^\s*/, '')
+      .replace(/\s*$/, '');
```

Comment:
this sucks

Reply:
@hjdivad the newline or the leading dot? Personally highly prefer the leading dot to the trailing dot (also pretty sure leading dot is overwhelmingly more common a style). Always putting things on new-lines though... 🤔

Reply:
The leading dot.

From my experience leading dot is slightly more popular than trailing dot.

Trailing dot is just better though: they both look about the same, leading dot has no benefits, trailing dot has two benefits, one minor and one significant.

The minor benefit is that it's more general because it doesn't rely on the language having as much lookahead.  This means that trailing dot works in more languages and so you don't have to switch styles if you switch between languages.

There's a related and more significant problem, which is that the newlines will be interpreted as statement delimiters when pasting code into a REPL.  It's pretty annoying that leading-dot code cannot be copy&pasted into a REPL.

---

What's your reason for preferring leading dot?  Is there a reason beyond aesthetics?

---

In any case it seems we can't configure prettier to use trailing dot.  As I said in my earlier comment, this isn't a blocker.  I'd rather have worse but automatic formatting, than manual formatting.